src/OAuth/Response/PathUserResponse.php

```diff
@@ -33,12 +33,30 @@ class PathUserResponse extends AbstractUserResponse
         'profilepicture' => null,
     ];
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getUserIdentifier(): string
+    {
+        $value = $this->getValueForPath('identifier');
+        if (!$value) {
```

Comment:
This will fail if the username is `'0'`, so perhaps it would be better to use `if (null === $value)` instead.

Reply:
In fact it was like that :)